corehq/apps/app_manager/templates/app_manager/partials/settings/app_settings.html

```diff
@@ -187,7 +187,11 @@ <h6>Example:</h6>
             </p>
             <a class="btn btn-primary"
                href="{% url 'pull_master_app' domain app.id %}">
-                {% trans 'Update from Master App' %}
+                {% trans 'Update from Master App now' %}
+            </a>
+            <a class="btn btn-primary"
```

Comment:
There should only be one `.btn-primary` on a page. Whichever of these is more common should use it, and the other one should use `.btn-default`.

Reply:
Cool. Waiting on feedback on keeping just one or both buttons from Product and then would change accordingly